server/database/config/db_build.sql

```diff
@@ -0,0 +1,67 @@
+BEGIN;
+  DROP TABLE IF EXISTS employees,questions,employees_questions,forms,employee_form,form_questions
+  CASCADE;
+
+
+
+CREATE TABLE employees
+(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50) NOT NULL,
+  email VARCHAR  ,
+  image TEXT,
```

Comment:
bytea is the data type for an image, why using TEXT? if you mean to use a URL as a source we can start as a bytea and store it as an image from the beginning

Reply:
this type has been modified.